topics/ecology/tutorials/Metashrimps_tutorial/tutorial.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: tutorial_hands_on
+
+title: Creating FAIR Quality assessment reports and draft of Data Papers from EML
+  metadata with MetaShRIMPS
+zenodo_link: https://doi.org/10.5281/zenodo.8130567
+questions:
+- How to improve the FAIR quality of an EML metadata ?
+- How to use metadata for machine actionnable processes ?
+- What is the point of having a FAIR metadata ?
+objectives:
+- Learn how to use the interactive tool Metashrimps
+- Understand the challenges MetaShRIMPS is trying to respond to
+- Learn how to create a FAIR Quality assessment report from a metadata using EML standard
+- Understand the concept of Data Paper and learn how to produce it
+- Explain the necessity of using such tools when producing ecological metadata
+time_estimation: 30mn
+key_points:
+- This tool aims to improve FAIR quality of metadata
+- Creating metadata as FAIR as possible is a must
+- Be carefull of the format and standard of metadata used only EML metadata will work
+tags:
+  - Metadata
+  - EML
+  - FAIR
+  - Data Paper
+contributors:
+- TanguyGen
+- yvanlebras
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.

@TanguyGen please also add your information to the CONTRIBUTORS.yaml so the tests can get a bit further :)